heroku-24-build/Dockerfile

```diff
@@ -8,3 +8,5 @@ USER 1002
 ENV CNB_USER_ID=1002
 ENV CNB_GROUP_ID=1000
 ENV CNB_STACK_ID "heroku-24"
+ENV CNB_TARGET_OS="linux"
+ENV CNB_TARGET_ARCH="amd64"
```

Comment:
Obviously, this is not correct for arm builds. This probably needs to be DOCKERFILE arg.

Reply:
Do we want to make this change now and then parameterize or leave it alone for now? I haven't looked how the build process is for this one.